core/state/snapshot/wipe.go

```diff
@@ -82,8 +83,9 @@ func wipeContent(db ethdb.KeyValueStore) error {
 }
 
 // wipeKeyRange deletes a range of keys from the database starting with prefix
-// and having a specific total key length.
-func wipeKeyRange(db ethdb.KeyValueStore, kind string, prefix []byte, keylen int) error {
+// and having a specific total key length. The start and limit is optional for
```

Comment:
Please document how start and limit works. E.g., afaict the `start + limit` is the first key that will _not_ be wiped.

Reply:
Oh and it's not `start` and `limit`, but `prefix` and `limit`, right?

Reply:
updated.